src/screens/Login/index.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+import LoginForm from '../../components/LoginForm';
+
+function Login() {
+  return <LoginForm />;
+}
+
+export default Login;
```

Comment:
this is a useless component, why not call `LoginForm` in the routes directly? SAME for register as i said in previous PR

Reply:
I prefer to have a component for each screen of the application